lib/backend/memory/memory.go

```diff
@@ -317,9 +320,6 @@ func (m *Memory) GetRange(ctx context.Context, startKey []byte, endKey []byte, l
 
 // KeepAlive updates TTL on the lease
 func (m *Memory) KeepAlive(ctx context.Context, lease backend.Lease, expires time.Time) error {
-	if lease.IsEmpty() {
-		return trace.BadParameter("lease is empty")
-	}
```

Comment:
The other operations on the memory backend check that the key is nonempty, we should probably do the same on `lease.Key` here (or remove the checks in the other operations, assuming that the btree can handle a key of length 0).